apps/concierge_site/lib/mail_templates/notification.txt.eex

```diff
@@ -11,6 +11,10 @@ The issue described below has ended:
 
 <%= notification.header %>
 
+<%= notification.image_url %>
```

Comment:
question: If these values are nil, does it result in an email with a bunch of extra empty lines? If so it might be worth including `if` statements.

Reply:
Ah, I put the "if" in the MJML but forgot it here. It results in an email with a size 0 image element but that's still not ideal.